src/scene/container/RenderGroup.ts

```diff
@@ -267,6 +258,21 @@ export class RenderGroup implements Instruction
         return out;
     }
 
+    /**
+     * @ignore
+     * @param viewContainer
+     * @deprecated - this method is not used and will be removed in the future
+     */
+    public updateRenderable(viewContainer: ViewContainer)
+    {
+        // only update if its visible!
+        if (viewContainer.globalDisplayStatus < 0b111) return;
+
+        viewContainer.didViewUpdate = false;
+        // actually updates the renderable..
+        this.instructionSet.renderPipes[viewContainer.renderPipeId].updateRenderable(viewContainer);
+    }
```

Comment:
what is the purpose of no longer calling this function?
seems like a needless breaking change and caused duplication of code between `RenderGroupSystem` and `updateRenderGroupTransform`